timetable/forms.py

```diff
@@ -160,3 +161,78 @@ def clean(self):
         #TODO check for encoding
         #TODO check for csv import errors
         return cleaned_data
+
+class SelectDateForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday)
+    
+    
+class AddDayPlanForm(Form):
+    date = Html5DateField(label=_('Date'), initial=get_next_schoolday())
```

Comment:
Here is an error:
get_next_schoolday() is a value. By adding parentheses You execute a code and get function value. The issue is that the currently initial is the value calculated during server startup. By dropping the parentheses, get_next_schoolday will be evaluated every request, returning the next school day after now (not after server's last restart)